versions.yaml

```diff
@@ -191,7 +191,7 @@ externals:
   attestation-agent:
     description: "Provide attested key unwrapping for image decryption"
     url: "https://github.com/confidential-containers/attestation-agent"
-    version: "v0.4.0"
+    version: "d7ace56"
```

Comment:
this needs to be a full-length hash, it doesn't work otherwise.

Reply:
Thanks, I've fixed it.